test/extensions/filters/http/compressor/compressor_filter_speed_test.cc

```diff
@@ -71,7 +71,39 @@ class MockZstdCompressorFactory : public Envoy::Compression::Compressor::Compres
   const uint64_t chunk_size_{4096};
 };
 
-using CompressionParams = std::tuple<int64_t, uint64_t, int64_t, uint64_t>;
+class MockBrotliCompressorFactory : public Envoy::Compression::Compressor::CompressorFactory {
+public:
+  MockBrotliCompressorFactory(uint32_t quality) : quality_(quality) {}
```

Comment:
explicit

Reply:
OK, I have changed to use default values in config for gzip and brotli, but there is no default value for zstd. Do you think if I can add some default values in zstd's config?